db/schema.rb

```diff
@@ -36,12 +36,11 @@
   end
 
   create_table "post_photos", force: :cascade do |t|
-    t.string    "photo",           limit: 255,                                              null: false
-    t.float     "image_direction",                                                          null: false
-    t.text      "note",                                                                     null: false
+    t.string    "photo",      limit: 255,                                              null: false
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.